direnvrc

```diff
@@ -1,5 +1,122 @@
 # shellcheck shell=bash
 
+_nix_import_env() {
+  local env=$1
+
+  local term_backup=$TERM path_backup=$PATH
+  if [[ -n ${TMPDIR+x} ]]; then
+    local tmp_backup=$TMPDIR
+  fi
+  local impure_ssl_cert_file=${SSL_CERT_FILE:-__UNSET__}
```

Comment:
this could be simplified with `local impure_ssl_cert_file=$SSL_CERT_FILE` and then use the `-z` test below

Reply:
or `local impure_ssl_cert_file=${SSL_CERT_FILE:-}` if `set -u` is being used actually

Reply:
How would distinguish an empty string from an unset one in this case?